cmd/empire/server.go

```diff
@@ -108,5 +108,16 @@ func newAuthenticator(c *cli.Context, e *empire.Empire) auth.Authenticator {
 		)
 	}
 
+	// After the user is authenticated, check their GitHub Team membership.
+	if teamId := c.String(FlagGithubTeam); teamId != "" {
```

Comment:
Minor, but can we change the casing from `Id` to `ID`. There's some doc from google about how it's not idiomatic to use camel casing for things like `ID` and we try to follow that.